src/Symfony/Component/Notifier/Event/NotificationEvents.php

```diff
@@ -23,7 +21,7 @@ class NotificationEvents
     private $events = [];
     private $transports = [];
 
-    public function add(MessageEvent $event): void
+    public function add(SentMessageEvent $event): void
```

Comment:
This is a BC break which is ok but needs to be written down to the changelog of the notifier component